frontend/src/components/__tests__/useClusterPoll.test.tsx

```diff
@@ -0,0 +1,57 @@
+import {renderHook} from '@testing-library/react-hooks'
```

Comment:
praise: great test, it's readable and easy to follow even if it's more complicated than your average test, well done!